ui/app/transitions.js

```diff
@@ -0,0 +1,9 @@
+export default function () {
+  // Add your transitions here, like:
+  //   this.transition(
+  //     this.fromRoute('people.index'),
+  //     this.toRoute('people.detail'),
+  //     this.use('toLeft'),
+  //     this.reverse('toRight')
+  //   );
```

Comment:
`blocking`: Are you sure you want to commit this?

Reply:
Good catch; this was an early abandoned idea